homeassistant/components/switchbot/cover.py

```diff
@@ -0,0 +1,149 @@
+"""Support for SwitchBot curtains."""
```

Comment:
This is adding a new platform, which is out of scope for this PR.

Please make a separate PR for the configuration flow and one for adding the cover.

Reply:
Hi @frenck ,

I have removed the cover platform as requested.